### PR TITLE
Scenario2 javaified.

### PR DESCRIPTION
**Pull request by bartosz-witkowski**

### CODE REVIEW

File: task-2/src/main/java/MultiQueue.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
+package task2.j;
+
+/*
+ * This file presents the abstract interface for a concurrent multi-queue implementation of a prioritiy queue.
+ *
+ * The queue will be used in a tight loop in a performant-relevant setting with multiple consumer and producer
```
Comment (ajakubauskas) [inline task-2/src/main/java/MultiQueue.java:6]: either _performant_ or _performance-relevant_

File: task-2/src/main/java/MultiQueue.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  /*
+   * If not closed inserts the element into the queue. If closed this operation does nothig.
+   */
+  public void insert(A element);
+
+  /*
+   * Best effort polling for the smallest element (with respect to Ordering) of the queue.
+   *
+   *   * it might return `null` as a normal operating procedure even if `isEmpty` == false or `size` is close.
+   *   * it might return an element that's not strictly the smallest one (with respect to Ordering)
```
Comment (ajakubauskas) [inline task-2/src/main/java/MultiQueue.java:52]: isn't `Ordering` just a scala thing? what would be java equivalent, `Comparator`?

Comment (bartosz-witkowski) [reply task-2/src/main/java/MultiQueue.java]: Good catch, reworded it so it uses natural ordering/Comparator like PriorityQueue

**Review by ajakubauskas — APPROVED** [review]
lgtm aside for a couple of nitpicks

**Pull request merged**